userspace/engine/falco_utils.cpp

```diff
@@ -151,7 +151,7 @@ std::string calculate_file_sha256sum(const std::string& filename)
 }
 #endif
 
-std::string sanitize_metric_name(const std::string& name)
+std::string sanitize_rule_name(const std::string& name)
```

Comment:
We still need this for the JSON output metrics even though we moved the Prometheus sanitization to libs. I renamed it to better reflect for what it is used.